client/components/sites-dropdown/index.jsx

```diff
@@ -38,20 +40,23 @@ export default class SitesDropdown extends PureComponent {
 		super( props );
 
 		this.selectSite = this.selectSite.bind( this );
-		this.toggleOpen = this.toggleOpen.bind( this );
+		this.toggleDropdown = this.toggleDropdown.bind( this );
 		this.onClose = this.onClose.bind( this );
 
-		const selectedSite = props.selectedSiteId
```

Comment:
We still need to depend on SitesList for primary site until we're a bit further along in the project.